lib/ansible/modules/packaging/language/pip.py

```diff
@@ -559,6 +571,7 @@ def main():
         absent=['uninstall', '-y'],
         latest=['install', '-U'],
         forcereinstall=['install', '-U', '--force-reinstall'],
+        downloaded=['download'],
```

Comment:
@robinro figured I'd revive this PR. notice that some of the tests that are failing are due to what I pointed out in the main PR comment that the download option is a newer one added to pip not too long ago. where would you prefer I check that download is an option? Probably makes the most sense to allow pip to be found first, then check if download is an option and fail if not.